pkgs/by-name/dh/dhcpig/package.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "dhcpig";
+  version = "1.6";
+
+  src = fetchFromGitHub {
+    owner = "kamorin";
+    repo = "DHCPig";
+    rev = version;
+    hash = "sha256-MquLChDuJe3DdkxxKV4W0o49IIt7Am+yuhdOqUqexS8=";
+  };
+
+  nativeBuildInputs = with python3.pkgs; [
+    setuptools
+    scapy
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    scapy
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp pig.py $out/bin/dhcpig
+  '';
+
+  meta = with lib; {
+    description = "DHCP exhaustion script written in python using scapy network library";
+    homepage = "https://github.com/kamorin/DHCPig";
```

Tools and bots could take the entry and create a link to the actual changelog for the commit messages and/or the PRs.

```suggestion
    homepage = "https://github.com/kamorin/DHCPig";
    changelog = "https://github.com/kamorin/DHCPig/releases/tag/${version}";
```